eras/conway/impl/test/Test/Cardano/Ledger/Conway/Scls.hs

```diff
@@ -0,0 +1,138 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE ImportQualifiedPost #-}
```

Comment:
Consistency in the codebase is key. We don't want to mix different styles in the same repo. (FTR. I am all in favor of the extension itself in general)
```suggestion
```